### PR TITLE
Update tutorial docs

### PR DESCRIPTION
**Pull request by stuartmcalpine**
Updated the docs to now match with the tutorial notebooks.

Also updated the CLI docs to include `dregs ls`

### CODE REVIEW

File: docs/source/tutorial_notebooks/getting_started.ipynb [hunk 1]
```diff
@@ -31,15 +23,7 @@
...
-    "\n",
-    "Also, make sure you have completed the one-time-setup for DREGS (more details [here](http://lsstdesc.org/dataregistry/installation.html)), namely:\n",
-    "\n",
-    "- You have added the database connection information in your `~/.config_reg_access` file\n",
-    "- You have added your database authentication information in your `~/.pgpass` file\n",
-    "\n",
-    "Finally, for this notebook to work for you interactively, you will need to be operating within a kernel that has the `dataregistry` package installed (see [here](https://docs.nersc.gov/services/jupyter/how-to-guides/) how to register your own Python environments as Jupyter kernels).\n",
-    "\n",
+    "If you haven't done so already, check out the [getting setup](file:///home/mcalpine/Documents/dataregistry/dataregistry/docs/build/html/tutorial_setup.html) page from the docs if you want to run this tutorial interactively.\n",
```
Comment (JoanneBogart) [inline docs/source/tutorial_notebooks/getting_started.ipynb:26]: I don't think you want a reference to what looks like a local file on your machine.

File: docs/source/tutorial_notebooks/pipline_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    "\n",
+    "### What we cover in this tutorial\n",
+    "\n",
+    "In this tutorial we will learn how to:\n",
+    "\n",
+    "- Register a series of dependant datasets from a pipeline into DREGS.\n",
+    "\n",
+    "### Before we begin\n",
+    "\n",
+    "If you haven't done so already, check out the [getting setup](file:///home/mcalpine/Documents/dataregistry/dataregistry/docs/build/html/tutorial_setup.html) page from the docs if you want to run this tutorial interactively.\n",
```
Comment (JoanneBogart) [inline docs/source/tutorial_notebooks/pipline_datasets.ipynb:34]: Same comment as above: this appears to be referencing a file on your local machine.

File: docs/source/tutorial_notebooks/pipline_datasets.ipynb [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "9337f001-5e7c-4141-a60c-5e99052aee3d",
+   "metadata": {},
+   "source": [
+    "<img src=\"../_static/DREGS_logo.png\" width=\"300\"/>\n",
+    "\n",
+    "# Working with pipeline datasets\n",
+    "\n",
+    "This tutorial focuses on how to register data into `DREGS` from a\n",
```
Comment (JoanneBogart) [inline docs/source/tutorial_notebooks/pipline_datasets.ipynb:12]: Did you really mean to call this notebook "pipline_datasets.ipynb"?   I would change to "pipeline_datasets.ipynb".

**Review by JoanneBogart — CHANGES_REQUESTED** [review]
Generally looks good but there are some things which should be addressed.

File: docs/source/tutorial_notebooks/getting_started.ipynb [hunk 4]
```diff
@@ -7,20 +7,12 @@
    "source": [
     "<img src=\"../_static/DREGS_logo.png\" width=\"300\"/>\n",
     "\n",
-    "# Welcome to the **DREGS** tutorial\n",
+    "# Getting started with **DREGS**\n",
```
[the remark this answers is not in the document]
Comment (stuartmcalpine) [reply docs/source/tutorial_notebooks/getting_started.ipynb:10]: I've removed all the output for now.

Will have a look to see how best to do rendered versions.

**Review by JoanneBogart — APPROVED** [review]
Looks fine now.
I think we'll have to redo the logo without "DREGS", but that can be in a separate PR.

**Pull request merged**